.travis.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         - make bin
         - sudo cp odo /usr/bin
         - oc login -u developer
-        - make test-cmp-e2e
+        - travis_wait make test-cmp-e2e
```

Comment:
sometimes it takes more than 20 min, so it might not help so, I would suggest use,

```
travis_wait 30
```

Ref: https://docs.travis-ci.com/user/common-build-problems/

Reply:
ok

Reply:
> sometimes it takes more than 20 min, so it might not help so, I would suggest use,

no, it doesn't matter how long the test takes, travis-ci check if the command is outputting messages, if there is no output for more than 10 mins it kills the job, `travis_wait`  increases that to 20. we shouldn't need more than 20. Job can still run for more than 20 mins regardless of this